x-pack/legacy/plugins/siem/public/pages/hosts/host_details.tsx

```diff
@@ -261,26 +265,29 @@ const HostDetailsComponent = pure<HostDetailsComponentProps>(
                       >
                         {({
                           events,
-                          loading,
                           id,
                           inspect,
+                          loading,
+                          loadPage,
+                          pageInfo,
                           refetch,
                           totalCount,
-                          pageInfo,
-                          loadMore,
                         }) => (
                           <EventsTableManage
+                            data={events!}
```

Comment:
Looks like this will work
```ts
data={events}
```

and another area we can remove the "bang" and gain back our type assertion which again shows that awesome PR's like this one doing clean ups along the way is resulting in better and better code.

The two places we have found this one which would be great to clean up with this already pretty cleaned up PR are here below. I tested both areas and they look to pass when the bangs are removed:

<img width="705" alt="Screen Shot 2019-07-19 at 1 18 16 PM" src="https://user-images.githubusercontent.com/1151048/61559879-fd33fd00-aa27-11e9-999b-d1335f971a78.png">